src/Command/RegisterChangedSchemasCommand.php

```diff
@@ -176,15 +199,20 @@ private function registerFiles(array $avroFiles, OutputInterface $output, array
             }
 
             try {
-                $schema = \AvroSchema::parse($localSchema);
-            } catch (\AvroSchemaParseException $e) {
+                $schema = AvroSchema::parse($localSchema);
+            } catch (AvroSchemaParseException $e) {
                 $output->writeln(sprintf('Skipping %s for now because %s', $schemaName, $e->getMessage()));
-                $failed[] = $avroFile;
+                $failed[] = $schemaName;
                 continue;
             }
 
             $this->schemaRegistryApi->createNewSchemaVersion($schema, $schemaName);
```

Comment:
i am not sure, maybe this already returns the new version on success in the result?

Reply:
No, checked, it returns just ID